docs/Account/methods.md

```diff
@@ -0,0 +1,284 @@
+  - signer(`Signer`, required): The signer instance that will need to be passed. 
+  - chainId (`ChainId` enum, required): The identifier for the blockchain network. (e.g., ChainId.POLYGON_MUMBAI).
+  - bundlerUrl (`string`, required) OR bundler (`IBundler`, required) : bundler url which will be internally used to create bundler instance or the bundler instance. Bundler instance can also be used if one wants to customise the bundler. Refer to bundler [integration](./../Bundler/integration.mdx) for more details on bundler.
+  - biconomyPaymasterApiKey(`string`) OR paymaster (`IPaymaster`): one can either pass paymaster API key or the paymaster instance.
+  - entryPointAddress (`string`): default entry point address will be used if not passed, otherwise the passed address will be used.
+  - defaultValidationModule (`BaseValidationModule`): The default validation module to start with ( ECDSA ).
+  - activeValidationModule (`BaseValidationModule`): The active validation module to start with ( either ECDSA or Multi-chain ).
+  - rpcUrl (`string`): RPC URL of the chain
+  - index (`number`): index to create multiple smart accounts for an EOA
+
+**Returns**
+
+- `smartAccount` (`object`): An instance of the Biconomy Smart Account V2.
+
+:::info
+Building on Chiliz Mainnet or the Spicy Testnet? Note that the entry point address on this is different as it was deployed by us on the Biconomy team. The address of the entry point is : [0x00000061FEfce24A79343c27127435286BB7A4E1](https://scan.chiliz.com/address/0x00000061FEfce24A79343c27127435286BB7A4E1/contracts#address-tabs)
+:::
+
+## Smart Account Get Methods
+
+### getAccountAddress( )
+
+This method is used to retrieve the account address associated with the `smartAccount` instance.
+
+**Usage**
+
+```jsx
+const address = await smartAccount.getAccountAddress();
+```
+
+**Returns**
+
+- address (`Promise<string>`): A Promise resolving to the account address associated with the smartAccount instance.
+
+### getNonce( )
+
+This method is used to retrieve the nonce associated with the `smartAccount` instance.
+
+**Usage**
+
+```jsx
+
+const nonce = await smartAccount.getNonce();
+console.log(nonce.toNumber());
+```
+**Parameters**
+- nonceKey(`number`): one can also pass the optional parameter nonceKey in the case of two-dimensional nonces
+
+**Returns**
+
+- address (`Promise<BigNumber>`): A Promise resolving to BigNumber indicating the nonce of the smart account.
+
+### index
+
+This is used to retrieve the index of the current active smart account.
+
+**Usage**
+```jsx
+const index = smartAccount.index;
+```
+
+**Returns**
+- index (`number`): A number indicating the index of current active smart account.
+
+## UserOp Methods
+
+### buildUserOp( )
+This method is used for configuring and setting up all properties of the `userOp` object.  It involves preparing and populating the userOp with necessary information and parameters required for execution by bundler.
```

Comment:
not all properties really.
it would not have signature
it could have paymasterAndData populated at this point in very limited use case of sponsorship pm for Biconomy.

There is no mention of batching.

Should be simple like converts a traditional transaction or batch of transactions into a partial user operation. 

Can also mention individual fields of userOp that gets populated.. initCode, sender, nonce, maxFeePerGas, maxPriorityFeePerGas, callGasLimit, verificationGasLimit, preVerificationGas [ As this step also involves internally estimating gas for the userOp ]